main/helpers.py

```diff
@@ -3,8 +3,8 @@
 
 def get_invalid_matches():
     matches = Match.objects.all()
-    invalid_matches = [match for match in matches if is_match_invalid(match)]
-    return invalid_matches
+    invalid_match_ids = [match.id for match in matches if is_match_invalid(match)]
+    return invalid_match_ids
```

Comment:
This was an interesting change. I'd expect this function to return matches and not match ids. I'd recommend returning matches, or renaming the function `get_invalid_match_ids`.

The first one is a bit more versatile.

Reply:
I originally was using matches, but I realised that if I wanted to print off the list of invalid_matches to the user once they had been removed, I wouldn't be able to access the match object seeing as it would no longer exist. So I decided to pass along the ids as those are just numbers.
I do agree the function name is confusing, I will change that